src/missing_deps.h

```diff
@@ -45,14 +45,15 @@ class MissingDependencyPrinter : public MissingDependencyScannerDelegate {
 };
 
 struct EdgePair {
-  Edge* from;
-  Edge* to;
+  EdgePair(Edge* from_, Edge* to_) : from(from_), to(to_) {}
```

Comment:
I'm not trying to nitpick you, sorry :(

I think that the typical Ninja coding conventions have the underscore for member variables, and the constructor parameters are underscore-less.

Reply:
I don't mind at all

Reply:
btw: There's no need to use a different name,  `EdgePair(Edge* from, Edge* to) : from(from), to(to) {}` works.

Reply:
yeah, I'm aware, but I really don't like relying on this particular nook of c++ name scoping